commit 4ad1f7e4d303ff2a2b2c2a485bfcf3884679a71e
AuthorDate: 2025-11-25 16:13:04 +0100
CommitDate: 2026-03-30 15:00:05 +0200
feat(modele-ti): nouvelles règles pour la réforme des cotisations


diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes"
index 1b634348c..6a5b6ce97 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations-et-contributions.publicodes"
@@ -53,3 +53,43 @@ indépendant . cotisations et contributions . formation professionnelle:
   références:
     Contribution à la formation professionnelle (CFP): https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/vos-cotisations.html#ancre-formation-professionnelle
     Contribution à la formation professionnelle (CFP) des entrepreneurs individuels: https://entreprendre.service-public.fr/vosdroits/F23459
+
+indépendant . cotisations et contributions . après réforme:
+  valeur: date > 01/2025
+
+indépendant . cotisations et contributions . assiette sociale:
+  description: |
+    Avec la réforme du calcul de l’assiette de cotisations et contributions
+    sociales des travailleurs indépendants portée par la Loi de Financement de la
+    Sécurité Sociale 2024,
+    **l’assiette des cotisations sociales est alignée sur l’assiette de la CSG-CRDS**.
+
+    Pour obtenir l’assiette des cotisations sociales à partir de celle de la
+    CSG-CRDS il faut :
+    - y ajouter les revenus de remplacement (hors affection de longue durée)
+    - et en soustraire la participation, l’intéressement et l’abondement au PER.
+
+  variations:
+    - si: après réforme
+      alors:
+        somme:
+          - CSG-CRDS . assiette
+          - IJSS . après abattement
+          - revenus étrangers . montant
+          # - (- cotisations facultatives . déductibles)
+    - sinon:
+        valeur: sans plancher
+        plancher: 0 €/an
+
+  avec:
+    sans plancher:
+      somme:
+        - revenu professionnel
+        - cotisations facultatives . déductibles
+
+  références:
+    Réforme de l’assiette sociale et du barème des cotisations sociales: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/reforme-cotisations-independants.html
+    Article L131-6 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000048683707
+    Article L3312-4 du Code du travail: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000037075048
+    Article L3324-5 du Code du travail: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000038589722
+    Article L3332-27 du Code du travail: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000037075039
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/ind\303\251pendant.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/ind\303\251pendant.publicodes"
index aa6178b1d..20c531f29 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/ind\303\251pendant.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/ind\303\251pendant.publicodes"
@@ -36,13 +36,47 @@ indépendant . revenu professionnel:
   unité: €/an
   résoudre la référence circulaire: oui
   variations:
-    - si: entreprise . imposition = 'IS'
+    - si: entreprise . imposition . IS
       alors:
         somme:
           - rémunération . nette
           - cotisations et contributions . non déductibles
     - sinon: entreprise . résultat fiscal
 
+indépendant . revenu brut:
+  description: |
+    Le revenu brut ou super-brut correspond au coût du travail ou encore au chiffre
+    d’affaires après déduction des charges professionnelles d’exploitations (hors
+    cotisations et contributions sociales).
+
+    ### À l’impôt sur le revenu
+
+    Le revenu brut correspond au
+    **chiffre d’affaires moins les charges** de l’entreprise liées à
+    l’exploitation, c’est-à-dire aux charges autres que les cotisations sociales,
+    la CSG et la CRDS.
+
+    Pour les gérantes et gérants associés, le chiffre d’affaires est proratisé.
+
+    ### À l’impôt sur les sociétés
+
+    Le revenu brut correspond à la **rémunération brute** et à une part des
+    dividendes. La rémunération prise en compte n’inclue pas la déduction des frais
+    professionnels, ni les cotisations sociales, la CSG et la CRDS.
+
+  note: |
+    Le moteur de calcul n’intègre pas encore les dividendes dans le revenu brut.
+
+  variations:
+    - si: entreprise . imposition . IR
+      alors: entreprise . chiffre d'affaires - entreprise . charges
+    - sinon:
+        somme:
+          - rémunération . totale
+          # - bénéficiaire . dividendes . bruts
+  références:
+    Réforme de l’assiette sociale et du barème des cotisations sociales: https://www.urssaf.fr/accueil/independant/comprendre-payer-cotisations/reforme-cotisations-independants.html
+
 indépendant . PSS proratisé:
   titre: plafond de la sécurité sociale proratisé
   description: |
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/revenus/IJSS.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/revenus/IJSS.publicodes"
index 73e049281..5aabfe29e 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/revenus/IJSS.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/revenus/IJSS.publicodes"
@@ -37,7 +37,6 @@ indépendant . IJSS . montant:
     - indemnité journalière forfaitaire d’interruption d’activité (maternité)
     - indemnité de remplacement pour maternité, paternité ou adoption
   par défaut: 0 €/an
-  unité: €/an
 
 indépendant . IJSS . imposable:
   titre: part imposable
@@ -60,3 +59,7 @@ indépendant . IJSS . imposable:
     - allocation journalière du proche aidant (AJPA, versée par la CAF)
   par défaut: 0 €/an
   unité: €/an
+
+indépendant . IJSS . après abattement:
+  valeur: IJSS . montant
+  abattement: 26%
